test/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.4.1
+RUN apt-get update && \
+    apt-get install -y jq && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN { \
+    echo "#!/bin/bash"; \
+    echo "eval \$(ssh-agent) >/dev/null"; \
+    echo "ssh-add /root/.ssh/id_rsa"; \
+    echo "exec \"\$@\""; \
+    } > /entrypoint.sh && chmod 755 /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
+
+RUN mkdir -p /root/.puppetlabs/bolt && \
+    echo "disabled: true" > /root/.puppetlabs/bolt/analytics.yaml
+
+COPY .cdpe-workflow-tests-config.json /root/
```

Comment:
is this with or without a leading '.'? Because in the WF tests for example, it doesn't have a dot. Also, what is the expected CWD for this to be an unqualified path, please?

Reply:
This is with a dot. The Makefile build target puts this file in place and then removes it after the build. Just like the maven test runner image/build target.

The CWD is determined by the final argument to the `docker build` command, which in our case is `..` (see Makefile).

Reply:
I didn't change the dot or not, that was already in place and specified in the README.md and genParams.rb

Reply:
that was the name of the item to pull from 1Password, not a file-system reference.

Reply:
I'm not quite sure what we're discussing here :)

But this is only relevant when a new image is being built, not when it's being used.

The script was already looking for `~/.cdpe-workfile-tests-config.json` so that's where I put the file in the image.

Reply:
Then we're good.